client/src/home/slice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
+  activeGamesPlayerID: '',
+  activeGames: {},
+};
+
+const homeSlice = createSlice({
+  name: 'home',
+  initialState,
+  reducers: {
+    refreshActiveGames(state, action) {
+      if (!action.payload.id) {
+        // what should we do when refreshing with an ID we do not expect?
+        throw `requires a playerID: got "${action.payload.id}"`;
```

Comment:
Hmmm I think reducers should _only_ update state. Need to look into this more, but my guess is throwing errors from a reducer is a no-no

Reply:
It may be a bit of a [gray area](https://redux.js.org/style-guide/style-guide#reducers-must-not-have-side-effects)

Under "Detailed Explanation":

> There are some gray areas to this rule. Strictly speaking, code such as console.log(state) is a side effect, but in practice has no effect on how the application behaves.

Reply:
I have no idea what I'm doing; don't trust me.